vic/drivers/cesm/src/get_global_param.c

```diff
@@ -0,0 +1,753 @@
+/******************************************************************************
+ * @section DESCRIPTION
+ *
+ * This routine reads the VIC model global control file, getting values for
+ * global parameters, model options, and debugging controls.
+ *
+ * @section LICENSE
+ *
+ * The Variable Infiltration Capacity (VIC) macroscale hydrological model
+ * Copyright (C) 2014 The Land Surface Hydrology Group, Department of Civil
+ * and Environmental Engineering, University of Washington.
+ *
+ * The VIC model is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *****************************************************************************/
+
+#include <vic_driver_cesm.h>
+
+/******************************************************************************
+ * @brief    Read the VIC model global control file, getting values for
+ *           global parameters, model options, and debugging controls.
+ *****************************************************************************/
+void
+get_global_param(FILE *gp)
+{
```

Comment:
sure, go ahead and open an issue.  I'm not sure its worth the work since every driver may have substantially different options but there certainly is some room to reduce redundant code.